src/types/lcp.ts

```diff
@@ -31,9 +31,14 @@ export interface LCPMetric extends Metric {
  */
 export interface LCPAttribution {
   /**
-   * The element corresponding to the largest contentful paint for the page.
+   * The selector of the element corresponding to the largest contentful paint
+   * for the page.
    */
   element?: string;
+  /**
+   * The element corresponding to the largest contentful paint for the page.
+   */
+  elementNode?: Node;
```

Comment:
I understand the desire to keep the API for `element` the same, but it does feel weird to call this a "node" when it's always going to be an element (and when the comment itself calls it an "element").

What do you think about using `target` and `targetElement` instead (for consistency with other interfaces)? I know that would be a breaking change, but a major version is the time to make such a change...

Reply:
Yeah I think it's time to take the plunge on this one! Done.

Though, personally I never liked the name "target" outside of interactions (which you "target" with your mouse or keyboard). But oh well. Consistency is more important.

Reply:
Reverting this, as no no longer saving the Node (so no need to report the node separately from the entry node).